test/integration/queue/aws/lambda/handler.js

```diff
@@ -75,6 +76,9 @@ describe('LambdaHandler', () => {
 
       await new LambdaHandler(arnToQueueInfo, handlerFactories)
         .handle(lambdaInput).catch(() => { });
+
```

Comment:
```suggestion
/* 
this sleep is necessary because testing with localstack sometimes the queue still has some messages 
that are in process of delete.
*/
await new Promise(resolve => setTimeout(resolve, 500));
```